form/database/migrations/2024_01_16_122809_create_products_table.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+
+    public function up(): void  
+    {
+        Schema::create('products', function (Blueprint $table) {
+            $table->id();
```

Comment:
@Yashbanu Can we change to `increments()` or `BigIncrements()` before that know the difference and add whichever suits best.

Reply:
@Naveen1895 sure